frontend/src/pages/NewAgreement.tsx

```diff
@@ -79,6 +82,21 @@ const NewAgreement: FC = () => {
     return <AgreementCreatedSuccessfully />;
   }
 
+  const handleClickOpenDialog = () => {
+    setOpenDialogState(true);
```

Comment:
@yishain11 can be a one liner

Reply:
this is an issue with prettier in my ide. is it realy important? to change it seems like a hassle...